pkg/sql/copy/copy_test: TestCopy failed

pkg/sql/copy/copy_test.TestCopy [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8728445?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8728445?buildTab=artifacts#/) on master @ [c245b153b4824f22447c80b3a25e745f423703de](https://github.com/cockroachdb/cockroach/commits/c245b153b4824f22447c80b3a25e745f423703de):


```
=== RUN   TestCopy
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/53491fdf16955d5896449a680358dad1/logTestCopy1584273836
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestCopy
    copy_test.go:146: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/53491fdf16955d5896449a680358dad1/logTestCopy1584273836
--- FAIL: TestCopy (9.35s)
=== RUN   TestCopy/copy_to_binary
    copy_test.go:88: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/694/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/copy/copy_test_/copy_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/copy/testdata/copy_to_binary:1:
        exec-ddl [0 args]
        CREATE TABLE t (id int primary key, t text);
        ----
    copy_test.go:88: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/694/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/copy/copy_test_/copy_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/copy/testdata/copy_to_binary:5:
        exec-ddl [0 args]
        INSERT INTO t VALUES
          (1, 'a tab' || chr(9) || ' separates us'),
          (2, 'some pipe || characters'),
          (3, 'new line chars!' || chr(10) || ' ok?'),
          (4, NULL);
        ----
    copy_test.go:88: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/694/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/copy/copy_test_/copy_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/copy/testdata/copy_to_binary:13:
         COPY t TO STDOUT BINARY
        expected:
        ERROR: unimplemented: binary format for COPY TO not implemented (SQLSTATE 0A000)
        HINT: You have attempted to use a feature that is not yet implemented.
        See: https://go.crdb.dev/issue-v/97180/v23.1
        
        found:
        ERROR: unimplemented: binary format for COPY TO not implemented (SQLSTATE 0A000)
        HINT: You have attempted to use a feature that is not yet implemented.
        See: https://go.crdb.dev/issue-v/97180/dev
    --- FAIL: TestCopy/copy_to_binary (2.15s)
```
<p>Parameters: <code>TAGS=bazel,gss,race</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/sql-queries
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestCopy.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-24585

pkg/sql/copy/copy_test.TestCopy [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8728441?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8728441?buildTab=artifacts#/) on master @ [c245b153b4824f22447c80b3a25e745f423703de](https://github.com/cockroachdb/cockroach/commits/c245b153b4824f22447c80b3a25e745f423703de):


```
=== RUN   TestCopy
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/53491fdf16955d5896449a680358dad1/logTestCopy1561320560
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestCopy
    copy_test.go:146: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/53491fdf16955d5896449a680358dad1/logTestCopy1561320560
--- FAIL: TestCopy (7.86s)
=== RUN   TestCopy/copy_to_binary
    copy_test.go:88: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3354/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/copy/copy_test_/copy_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/copy/testdata/copy_to_binary:1:
        exec-ddl [0 args]
        CREATE TABLE t (id int primary key, t text);
        ----
    copy_test.go:88: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3354/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/copy/copy_test_/copy_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/copy/testdata/copy_to_binary:5:
        exec-ddl [0 args]
        INSERT INTO t VALUES
          (1, 'a tab' || chr(9) || ' separates us'),
          (2, 'some pipe || characters'),
          (3, 'new line chars!' || chr(10) || ' ok?'),
          (4, NULL);
        ----
    copy_test.go:88: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3354/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/copy/copy_test_/copy_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/copy/testdata/copy_to_binary:13:
         COPY t TO STDOUT BINARY
        expected:
        ERROR: unimplemented: binary format for COPY TO not implemented (SQLSTATE 0A000)
        HINT: You have attempted to use a feature that is not yet implemented.
        See: https://go.crdb.dev/issue-v/97180/v23.1
        
        found:
        ERROR: unimplemented: binary format for COPY TO not implemented (SQLSTATE 0A000)
        HINT: You have attempted to use a feature that is not yet implemented.
        See: https://go.crdb.dev/issue-v/97180/dev
    --- FAIL: TestCopy/copy_to_binary (1.73s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestCopy.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/sql/copy/copy_test.TestCopy [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8728436?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8728436?buildTab=artifacts#/) on master @ [c245b153b4824f22447c80b3a25e745f423703de](https://github.com/cockroachdb/cockroach/commits/c245b153b4824f22447c80b3a25e745f423703de):


```
=== RUN   TestCopy
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/53491fdf16955d5896449a680358dad1/logTestCopy1617685289
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestCopy
    copy_test.go:146: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/53491fdf16955d5896449a680358dad1/logTestCopy1617685289
--- FAIL: TestCopy (1.90s)
=== RUN   TestCopy/copy_to_binary
    copy_test.go:88: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2547/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/copy/copy_test_/copy_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/copy/testdata/copy_to_binary:1:
        exec-ddl [0 args]
        CREATE TABLE t (id int primary key, t text);
        ----
    copy_test.go:88: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2547/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/copy/copy_test_/copy_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/copy/testdata/copy_to_binary:5:
        exec-ddl [0 args]
        INSERT INTO t VALUES
          (1, 'a tab' || chr(9) || ' separates us'),
          (2, 'some pipe || characters'),
          (3, 'new line chars!' || chr(10) || ' ok?'),
          (4, NULL);
        ----
    copy_test.go:88: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2547/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/copy/copy_test_/copy_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/copy/testdata/copy_to_binary:13:
         COPY t TO STDOUT BINARY
        expected:
        ERROR: unimplemented: binary format for COPY TO not implemented (SQLSTATE 0A000)
        HINT: You have attempted to use a feature that is not yet implemented.
        See: https://go.crdb.dev/issue-v/97180/v23.1
        
        found:
        ERROR: unimplemented: binary format for COPY TO not implemented (SQLSTATE 0A000)
        HINT: You have attempted to use a feature that is not yet implemented.
        See: https://go.crdb.dev/issue-v/97180/dev
    --- FAIL: TestCopy/copy_to_binary (0.36s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestCopy.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/sql/copy/copy_test.TestCopy [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8742828?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8742828?buildTab=artifacts#/) on master @ [7e2df35a2f6bf7a859bb0539c8ca43c4e72ed260](https://github.com/cockroachdb/cockroach/commits/7e2df35a2f6bf7a859bb0539c8ca43c4e72ed260):


```
=== RUN   TestCopy
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/53491fdf16955d5896449a680358dad1/logTestCopy2033903719
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestCopy
    copy_test.go:146: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/53491fdf16955d5896449a680358dad1/logTestCopy2033903719
--- FAIL: TestCopy (7.69s)
=== RUN   TestCopy/copy_to_binary
    copy_test.go:88: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1731/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/copy/copy_test_/copy_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/copy/testdata/copy_to_binary:1:
        exec-ddl [0 args]
        CREATE TABLE t (id int primary key, t text);
        ----
    copy_test.go:88: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1731/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/copy/copy_test_/copy_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/copy/testdata/copy_to_binary:5:
        exec-ddl [0 args]
        INSERT INTO t VALUES
          (1, 'a tab' || chr(9) || ' separates us'),
          (2, 'some pipe || characters'),
          (3, 'new line chars!' || chr(10) || ' ok?'),
          (4, NULL);
        ----
    copy_test.go:88: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1731/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/copy/copy_test_/copy_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/copy/testdata/copy_to_binary:13:
         COPY t TO STDOUT BINARY
        expected:
        ERROR: unimplemented: binary format for COPY TO not implemented (SQLSTATE 0A000)
        HINT: You have attempted to use a feature that is not yet implemented.
        See: https://go.crdb.dev/issue-v/97180/v23.1
        
        found:
        ERROR: unimplemented: binary format for COPY TO not implemented (SQLSTATE 0A000)
        HINT: You have attempted to use a feature that is not yet implemented.
        See: https://go.crdb.dev/issue-v/97180/dev
    --- FAIL: TestCopy/copy_to_binary (1.75s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestCopy.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/sql/copy/copy_test.TestCopy [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8742829?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8742829?buildTab=artifacts#/) on master @ [7e2df35a2f6bf7a859bb0539c8ca43c4e72ed260](https://github.com/cockroachdb/cockroach/commits/7e2df35a2f6bf7a859bb0539c8ca43c4e72ed260):


```
=== RUN   TestCopy
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/53491fdf16955d5896449a680358dad1/logTestCopy2444717766
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestCopy
    copy_test.go:146: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/53491fdf16955d5896449a680358dad1/logTestCopy2444717766
--- FAIL: TestCopy (9.13s)
=== RUN   TestCopy/copy_to_binary
    copy_test.go:88: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1355/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/copy/copy_test_/copy_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/copy/testdata/copy_to_binary:1:
        exec-ddl [0 args]
        CREATE TABLE t (id int primary key, t text);
        ----
    copy_test.go:88: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1355/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/copy/copy_test_/copy_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/copy/testdata/copy_to_binary:5:
        exec-ddl [0 args]
        INSERT INTO t VALUES
          (1, 'a tab' || chr(9) || ' separates us'),
          (2, 'some pipe || characters'),
          (3, 'new line chars!' || chr(10) || ' ok?'),
          (4, NULL);
        ----
    copy_test.go:88: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1355/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/copy/copy_test_/copy_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/copy/testdata/copy_to_binary:13:
         COPY t TO STDOUT BINARY
        expected:
        ERROR: unimplemented: binary format for COPY TO not implemented (SQLSTATE 0A000)
        HINT: You have attempted to use a feature that is not yet implemented.
        See: https://go.crdb.dev/issue-v/97180/v23.1
        
        found:
        ERROR: unimplemented: binary format for COPY TO not implemented (SQLSTATE 0A000)
        HINT: You have attempted to use a feature that is not yet implemented.
        See: https://go.crdb.dev/issue-v/97180/dev
    --- FAIL: TestCopy/copy_to_binary (2.11s)
```
<p>Parameters: <code>TAGS=bazel,gss,race</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestCopy.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/sql/copy/copy_test.TestCopy [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8742823?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8742823?buildTab=artifacts#/) on master @ [7e2df35a2f6bf7a859bb0539c8ca43c4e72ed260](https://github.com/cockroachdb/cockroach/commits/7e2df35a2f6bf7a859bb0539c8ca43c4e72ed260):


```
=== RUN   TestCopy
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/53491fdf16955d5896449a680358dad1/logTestCopy3828869063
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestCopy
    copy_test.go:146: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/53491fdf16955d5896449a680358dad1/logTestCopy3828869063
--- FAIL: TestCopy (2.07s)
=== RUN   TestCopy/copy_to_binary
    copy_test.go:88: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2679/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/copy/copy_test_/copy_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/copy/testdata/copy_to_binary:1:
        exec-ddl [0 args]
        CREATE TABLE t (id int primary key, t text);
        ----
    copy_test.go:88: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2679/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/copy/copy_test_/copy_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/copy/testdata/copy_to_binary:5:
        exec-ddl [0 args]
        INSERT INTO t VALUES
          (1, 'a tab' || chr(9) || ' separates us'),
          (2, 'some pipe || characters'),
          (3, 'new line chars!' || chr(10) || ' ok?'),
          (4, NULL);
        ----
    copy_test.go:88: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2679/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/copy/copy_test_/copy_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/copy/testdata/copy_to_binary:13:
         COPY t TO STDOUT BINARY
        expected:
        ERROR: unimplemented: binary format for COPY TO not implemented (SQLSTATE 0A000)
        HINT: You have attempted to use a feature that is not yet implemented.
        See: https://go.crdb.dev/issue-v/97180/v23.1
        
        found:
        ERROR: unimplemented: binary format for COPY TO not implemented (SQLSTATE 0A000)
        HINT: You have attempted to use a feature that is not yet implemented.
        See: https://go.crdb.dev/issue-v/97180/dev
    --- FAIL: TestCopy/copy_to_binary (0.50s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestCopy.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/sql/copy/copy_test.TestCopy [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8753025?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8753025?buildTab=artifacts#/) on master @ [c95bef097bd4c213c6b5c0c125a9a846c4479d73](https://github.com/cockroachdb/cockroach/commits/c95bef097bd4c213c6b5c0c125a9a846c4479d73):


```
=== RUN   TestCopy
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/53491fdf16955d5896449a680358dad1/logTestCopy1172793821
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestCopy
    copy_test.go:146: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/53491fdf16955d5896449a680358dad1/logTestCopy1172793821
--- FAIL: TestCopy (2.63s)
=== RUN   TestCopy/copy_to_binary
    copy_test.go:88: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1713/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/copy/copy_test_/copy_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/copy/testdata/copy_to_binary:1:
        exec-ddl [0 args]
        CREATE TABLE t (id int primary key, t text);
        ----
    copy_test.go:88: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1713/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/copy/copy_test_/copy_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/copy/testdata/copy_to_binary:5:
        exec-ddl [0 args]
        INSERT INTO t VALUES
          (1, 'a tab' || chr(9) || ' separates us'),
          (2, 'some pipe || characters'),
          (3, 'new line chars!' || chr(10) || ' ok?'),
          (4, NULL);
        ----
    copy_test.go:88: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1713/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/copy/copy_test_/copy_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/copy/testdata/copy_to_binary:13:
         COPY t TO STDOUT BINARY
        expected:
        ERROR: unimplemented: binary format for COPY TO not implemented (SQLSTATE 0A000)
        HINT: You have attempted to use a feature that is not yet implemented.
        See: https://go.crdb.dev/issue-v/97180/v23.1
        
        found:
        ERROR: unimplemented: binary format for COPY TO not implemented (SQLSTATE 0A000)
        HINT: You have attempted to use a feature that is not yet implemented.
        See: https://go.crdb.dev/issue-v/97180/dev
    --- FAIL: TestCopy/copy_to_binary (0.57s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestCopy.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/sql/copy/copy_test.TestCopy [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8753030?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8753030?buildTab=artifacts#/) on master @ [c95bef097bd4c213c6b5c0c125a9a846c4479d73](https://github.com/cockroachdb/cockroach/commits/c95bef097bd4c213c6b5c0c125a9a846c4479d73):


```
=== RUN   TestCopy
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/53491fdf16955d5896449a680358dad1/logTestCopy2413398462
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestCopy
    copy_test.go:146: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/53491fdf16955d5896449a680358dad1/logTestCopy2413398462
--- FAIL: TestCopy (10.06s)
=== RUN   TestCopy/copy_to_binary
    copy_test.go:88: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2511/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/copy/copy_test_/copy_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/copy/testdata/copy_to_binary:1:
        exec-ddl [0 args]
        CREATE TABLE t (id int primary key, t text);
        ----
    copy_test.go:88: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2511/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/copy/copy_test_/copy_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/copy/testdata/copy_to_binary:5:
        exec-ddl [0 args]
        INSERT INTO t VALUES
          (1, 'a tab' || chr(9) || ' separates us'),
          (2, 'some pipe || characters'),
          (3, 'new line chars!' || chr(10) || ' ok?'),
          (4, NULL);
        ----
    copy_test.go:88: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2511/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/copy/copy_test_/copy_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/copy/testdata/copy_to_binary:13:
         COPY t TO STDOUT BINARY
        expected:
        ERROR: unimplemented: binary format for COPY TO not implemented (SQLSTATE 0A000)
        HINT: You have attempted to use a feature that is not yet implemented.
        See: https://go.crdb.dev/issue-v/97180/v23.1
        
        found:
        ERROR: unimplemented: binary format for COPY TO not implemented (SQLSTATE 0A000)
        HINT: You have attempted to use a feature that is not yet implemented.
        See: https://go.crdb.dev/issue-v/97180/dev
    --- FAIL: TestCopy/copy_to_binary (2.27s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestCopy.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/sql/copy/copy_test.TestCopy [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8753033?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8753033?buildTab=artifacts#/) on master @ [c95bef097bd4c213c6b5c0c125a9a846c4479d73](https://github.com/cockroachdb/cockroach/commits/c95bef097bd4c213c6b5c0c125a9a846c4479d73):


```
=== RUN   TestCopy
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/53491fdf16955d5896449a680358dad1/logTestCopy48866363
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestCopy
    copy_test.go:146: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/53491fdf16955d5896449a680358dad1/logTestCopy48866363
--- FAIL: TestCopy (11.43s)
=== RUN   TestCopy/copy_to_binary
    copy_test.go:88: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2030/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/copy/copy_test_/copy_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/copy/testdata/copy_to_binary:1:
        exec-ddl [0 args]
        CREATE TABLE t (id int primary key, t text);
        ----
    copy_test.go:88: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2030/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/copy/copy_test_/copy_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/copy/testdata/copy_to_binary:5:
        exec-ddl [0 args]
        INSERT INTO t VALUES
          (1, 'a tab' || chr(9) || ' separates us'),
          (2, 'some pipe || characters'),
          (3, 'new line chars!' || chr(10) || ' ok?'),
          (4, NULL);
        ----
    copy_test.go:88: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2030/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/copy/copy_test_/copy_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/copy/testdata/copy_to_binary:13:
         COPY t TO STDOUT BINARY
        expected:
        ERROR: unimplemented: binary format for COPY TO not implemented (SQLSTATE 0A000)
        HINT: You have attempted to use a feature that is not yet implemented.
        See: https://go.crdb.dev/issue-v/97180/v23.1
        
        found:
        ERROR: unimplemented: binary format for COPY TO not implemented (SQLSTATE 0A000)
        HINT: You have attempted to use a feature that is not yet implemented.
        See: https://go.crdb.dev/issue-v/97180/dev
    --- FAIL: TestCopy/copy_to_binary (2.59s)
```
<p>Parameters: <code>TAGS=bazel,gss,race</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestCopy.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/sql/copy/copy_test.TestCopy [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8760435?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8760435?buildTab=artifacts#/) on master @ [3d054f37c7c87f53cb56fac4e5500f0d1130d09a](https://github.com/cockroachdb/cockroach/commits/3d054f37c7c87f53cb56fac4e5500f0d1130d09a):


```
=== RUN   TestCopy
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/53491fdf16955d5896449a680358dad1/logTestCopy3137157368
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestCopy
    copy_test.go:146: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/53491fdf16955d5896449a680358dad1/logTestCopy3137157368
--- FAIL: TestCopy (9.98s)
=== RUN   TestCopy/copy_to_binary
    copy_test.go:88: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1888/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/copy/copy_test_/copy_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/copy/testdata/copy_to_binary:1:
        exec-ddl [0 args]
        CREATE TABLE t (id int primary key, t text);
        ----
    copy_test.go:88: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1888/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/copy/copy_test_/copy_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/copy/testdata/copy_to_binary:5:
        exec-ddl [0 args]
        INSERT INTO t VALUES
          (1, 'a tab' || chr(9) || ' separates us'),
          (2, 'some pipe || characters'),
          (3, 'new line chars!' || chr(10) || ' ok?'),
          (4, NULL);
        ----
    copy_test.go:88: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1888/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/copy/copy_test_/copy_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/copy/testdata/copy_to_binary:13:
         COPY t TO STDOUT BINARY
        expected:
        ERROR: unimplemented: binary format for COPY TO not implemented (SQLSTATE 0A000)
        HINT: You have attempted to use a feature that is not yet implemented.
        See: https://go.crdb.dev/issue-v/97180/v23.1
        
        found:
        ERROR: unimplemented: binary format for COPY TO not implemented (SQLSTATE 0A000)
        HINT: You have attempted to use a feature that is not yet implemented.
        See: https://go.crdb.dev/issue-v/97180/dev
    --- FAIL: TestCopy/copy_to_binary (2.37s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestCopy.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/sql/copy/copy_test.TestCopy [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8760431?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8760431?buildTab=artifacts#/) on master @ [3d054f37c7c87f53cb56fac4e5500f0d1130d09a](https://github.com/cockroachdb/cockroach/commits/3d054f37c7c87f53cb56fac4e5500f0d1130d09a):


```
=== RUN   TestCopy
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/53491fdf16955d5896449a680358dad1/logTestCopy3471669770
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestCopy
    copy_test.go:146: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/53491fdf16955d5896449a680358dad1/logTestCopy3471669770
--- FAIL: TestCopy (2.08s)
=== RUN   TestCopy/copy_to_binary
    copy_test.go:88: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2430/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/copy/copy_test_/copy_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/copy/testdata/copy_to_binary:1:
        exec-ddl [0 args]
        CREATE TABLE t (id int primary key, t text);
        ----
    copy_test.go:88: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2430/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/copy/copy_test_/copy_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/copy/testdata/copy_to_binary:5:
        exec-ddl [0 args]
        INSERT INTO t VALUES
          (1, 'a tab' || chr(9) || ' separates us'),
          (2, 'some pipe || characters'),
          (3, 'new line chars!' || chr(10) || ' ok?'),
          (4, NULL);
        ----
    copy_test.go:88: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2430/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/copy/copy_test_/copy_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/copy/testdata/copy_to_binary:13:
         COPY t TO STDOUT BINARY
        expected:
        ERROR: unimplemented: binary format for COPY TO not implemented (SQLSTATE 0A000)
        HINT: You have attempted to use a feature that is not yet implemented.
        See: https://go.crdb.dev/issue-v/97180/v23.1
        
        found:
        ERROR: unimplemented: binary format for COPY TO not implemented (SQLSTATE 0A000)
        HINT: You have attempted to use a feature that is not yet implemented.
        See: https://go.crdb.dev/issue-v/97180/dev
    --- FAIL: TestCopy/copy_to_binary (0.45s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestCopy.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/sql/copy/copy_test.TestCopy [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8760437?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8760437?buildTab=artifacts#/) on master @ [3d054f37c7c87f53cb56fac4e5500f0d1130d09a](https://github.com/cockroachdb/cockroach/commits/3d054f37c7c87f53cb56fac4e5500f0d1130d09a):


```
=== RUN   TestCopy
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/53491fdf16955d5896449a680358dad1/logTestCopy4267532780
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestCopy
    copy_test.go:146: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/53491fdf16955d5896449a680358dad1/logTestCopy4267532780
--- FAIL: TestCopy (11.63s)
=== RUN   TestCopy/copy_to_binary
    copy_test.go:88: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3487/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/copy/copy_test_/copy_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/copy/testdata/copy_to_binary:1:
        exec-ddl [0 args]
        CREATE TABLE t (id int primary key, t text);
        ----
    copy_test.go:88: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3487/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/copy/copy_test_/copy_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/copy/testdata/copy_to_binary:5:
        exec-ddl [0 args]
        INSERT INTO t VALUES
          (1, 'a tab' || chr(9) || ' separates us'),
          (2, 'some pipe || characters'),
          (3, 'new line chars!' || chr(10) || ' ok?'),
          (4, NULL);
        ----
    copy_test.go:88: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3487/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/copy/copy_test_/copy_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/copy/testdata/copy_to_binary:13:
         COPY t TO STDOUT BINARY
        expected:
        ERROR: unimplemented: binary format for COPY TO not implemented (SQLSTATE 0A000)
        HINT: You have attempted to use a feature that is not yet implemented.
        See: https://go.crdb.dev/issue-v/97180/v23.1
        
        found:
        ERROR: unimplemented: binary format for COPY TO not implemented (SQLSTATE 0A000)
        HINT: You have attempted to use a feature that is not yet implemented.
        See: https://go.crdb.dev/issue-v/97180/dev
    --- FAIL: TestCopy/copy_to_binary (2.68s)
```
<p>Parameters: <code>TAGS=bazel,gss,race</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestCopy.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/sql/copy/copy_test.TestCopy [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8766972?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8766972?buildTab=artifacts#/) on master @ [e882d50c9b9d15a4d5cc88a0009724b9837f8630](https://github.com/cockroachdb/cockroach/commits/e882d50c9b9d15a4d5cc88a0009724b9837f8630):


```
=== RUN   TestCopy
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/53491fdf16955d5896449a680358dad1/logTestCopy4081408154
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestCopy
    copy_test.go:146: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/53491fdf16955d5896449a680358dad1/logTestCopy4081408154
--- FAIL: TestCopy (2.37s)
=== RUN   TestCopy/copy_to_binary
    copy_test.go:88: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1023/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/copy/copy_test_/copy_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/copy/testdata/copy_to_binary:1:
        exec-ddl [0 args]
        CREATE TABLE t (id int primary key, t text);
        ----
    copy_test.go:88: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1023/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/copy/copy_test_/copy_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/copy/testdata/copy_to_binary:5:
        exec-ddl [0 args]
        INSERT INTO t VALUES
          (1, 'a tab' || chr(9) || ' separates us'),
          (2, 'some pipe || characters'),
          (3, 'new line chars!' || chr(10) || ' ok?'),
          (4, NULL);
        ----
    copy_test.go:88: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1023/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/copy/copy_test_/copy_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/copy/testdata/copy_to_binary:13:
         COPY t TO STDOUT BINARY
        expected:
        ERROR: unimplemented: binary format for COPY TO not implemented (SQLSTATE 0A000)
        HINT: You have attempted to use a feature that is not yet implemented.
        See: https://go.crdb.dev/issue-v/97180/v23.1
        
        found:
        ERROR: unimplemented: binary format for COPY TO not implemented (SQLSTATE 0A000)
        HINT: You have attempted to use a feature that is not yet implemented.
        See: https://go.crdb.dev/issue-v/97180/dev
    --- FAIL: TestCopy/copy_to_binary (0.49s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestCopy.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/sql/copy/copy_test.TestCopy [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8766977?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8766977?buildTab=artifacts#/) on master @ [e882d50c9b9d15a4d5cc88a0009724b9837f8630](https://github.com/cockroachdb/cockroach/commits/e882d50c9b9d15a4d5cc88a0009724b9837f8630):


```
=== RUN   TestCopy
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/53491fdf16955d5896449a680358dad1/logTestCopy3446516429
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestCopy
    copy_test.go:146: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/53491fdf16955d5896449a680358dad1/logTestCopy3446516429
--- FAIL: TestCopy (9.56s)
=== RUN   TestCopy/copy_to_binary
    copy_test.go:88: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3288/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/copy/copy_test_/copy_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/copy/testdata/copy_to_binary:1:
        exec-ddl [0 args]
        CREATE TABLE t (id int primary key, t text);
        ----
    copy_test.go:88: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3288/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/copy/copy_test_/copy_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/copy/testdata/copy_to_binary:5:
        exec-ddl [0 args]
        INSERT INTO t VALUES
          (1, 'a tab' || chr(9) || ' separates us'),
          (2, 'some pipe || characters'),
          (3, 'new line chars!' || chr(10) || ' ok?'),
          (4, NULL);
        ----
    copy_test.go:88: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3288/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/copy/copy_test_/copy_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/copy/testdata/copy_to_binary:13:
         COPY t TO STDOUT BINARY
        expected:
        ERROR: unimplemented: binary format for COPY TO not implemented (SQLSTATE 0A000)
        HINT: You have attempted to use a feature that is not yet implemented.
        See: https://go.crdb.dev/issue-v/97180/v23.1
        
        found:
        ERROR: unimplemented: binary format for COPY TO not implemented (SQLSTATE 0A000)
        HINT: You have attempted to use a feature that is not yet implemented.
        See: https://go.crdb.dev/issue-v/97180/dev
    --- FAIL: TestCopy/copy_to_binary (2.14s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestCopy.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/sql/copy/copy_test.TestCopy [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8766978?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8766978?buildTab=artifacts#/) on master @ [e882d50c9b9d15a4d5cc88a0009724b9837f8630](https://github.com/cockroachdb/cockroach/commits/e882d50c9b9d15a4d5cc88a0009724b9837f8630):


```
=== RUN   TestCopy
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/53491fdf16955d5896449a680358dad1/logTestCopy2527042066
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestCopy
    copy_test.go:146: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/53491fdf16955d5896449a680358dad1/logTestCopy2527042066
--- FAIL: TestCopy (12.09s)
=== RUN   TestCopy/copy_to_binary
    copy_test.go:88: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3460/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/copy/copy_test_/copy_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/copy/testdata/copy_to_binary:1:
        exec-ddl [0 args]
        CREATE TABLE t (id int primary key, t text);
        ----
    copy_test.go:88: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3460/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/copy/copy_test_/copy_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/copy/testdata/copy_to_binary:5:
        exec-ddl [0 args]
        INSERT INTO t VALUES
          (1, 'a tab' || chr(9) || ' separates us'),
          (2, 'some pipe || characters'),
          (3, 'new line chars!' || chr(10) || ' ok?'),
          (4, NULL);
        ----
    copy_test.go:88: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3460/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/copy/copy_test_/copy_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/copy/testdata/copy_to_binary:13:
         COPY t TO STDOUT BINARY
        expected:
        ERROR: unimplemented: binary format for COPY TO not implemented (SQLSTATE 0A000)
        HINT: You have attempted to use a feature that is not yet implemented.
        See: https://go.crdb.dev/issue-v/97180/v23.1
        
        found:
        ERROR: unimplemented: binary format for COPY TO not implemented (SQLSTATE 0A000)
        HINT: You have attempted to use a feature that is not yet implemented.
        See: https://go.crdb.dev/issue-v/97180/dev
    --- FAIL: TestCopy/copy_to_binary (2.79s)
```
<p>Parameters: <code>TAGS=bazel,gss,race</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestCopy.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/sql/copy/copy_test.go
+++ b/pkg/sql/copy/copy_test.go
@@ -18,6 +18,7 @@ import (
 	"fmt"
 	"io"
 	"net/url"
+	"regexp"
 	"runtime/pprof"
 	"strings"
 	"testing"
@@ -145,6 +146,8 @@ func TestCopy(t *testing.T) {
 	})
 }
 
+var issueLinkRE = regexp.MustCompile("https://go.crdb.dev/issue-v/([0-9]+)/.*")
+
 func expandErrorString(err error) string {
 	var sb strings.Builder
 	sb.WriteString(err.Error())
@@ -157,7 +160,7 @@ func expandErrorString(err error) string {
 			sb.WriteString(fmt.Sprintf("\nDETAIL: %s", pgErr.Detail))
 		}
 	}
-	return sb.String()
+	return issueLinkRE.ReplaceAllString(sb.String(), `https://go.crdb.dev/issue-v/$1/`)
 }
 
 // TestCopyFromTransaction tests that copy from rows are written with

--- a/pkg/sql/copy/testdata/copy_to_binary
+++ b/pkg/sql/copy/testdata/copy_to_binary
@@ -15,4 +15,4 @@ COPY t TO STDOUT BINARY
 ----
 ERROR: unimplemented: binary format for COPY TO not implemented (SQLSTATE 0A000)
 HINT: You have attempted to use a feature that is not yet implemented.
-See: https://go.crdb.dev/issue-v/97180/v23.1
+See: https://go.crdb.dev/issue-v/97180/

